data/resources/ui/pod/creation-page.ui

```diff
@@ -7,180 +7,350 @@
+                                                            <property name="valign">center</property>
+                                                          </object>
+                                                        </child>
+                                                      </object>
+                                                    </child>
+
+                                                    <child>
+                                                      <object class="AdwEntryRow" id="infra_common_pid_file_entry_row">
+                                                        <property name="activates-default">True</property>
+                                                        <property name="title" translatable="yes">Common PID File</property>
+                                                      </object>
+                                                    </child>
+
+                                                  </object>
+                                                </child>
+
+                                                <child>
+                                                  <object class="AdwPreferencesGroup">
+
+                                                    <child>
+                                                      <object class="AdwEntryRow" id="infra_command_entry_row">
+                                                        <property name="activates-default">True</property>
+                                                        <property name="title" translatable="yes">Command</property>
+                                                      </object>
+                                                    </child>
+
+                                                  </object>
+                                                </child>
+
+                                                <child>
+                                                  <object class="AdwPreferencesGroup">
+                                                    <property name="title" translatable="yes">Arguments</property>
+                                                      <child>
+                                                        <object class="GtkListBox" id="infra_command_arg_list_box">
+                                                          <style>
+                                                            <class name="boxed-list" />
+                                                          </style>
+                                                        </object>
+                                                      </child>
+                                                  </object>
+                                                </child>
+
+                                              </object>
+                                            </child>
+
+                                      </object>
```

Comment:
Indentation is not correct here.
```suggestion
                                          </child>

                                      </object>
```

Reply:
I moved both children to match the indentation (not 100% sure if this is what you meant)

Reply:
Yes exactly that. The block was way too big to make a full suggestion.